custom-example/custom.pri

```diff
@@ -17,8 +17,8 @@ win32 {
 }
 CUSTOM_QGC_VERSION = $${CUSTOM_QGC_VER_MAJOR}.$${CUSTOM_QGC_VER_MINOR}.$${CUSTOM_QGC_VER_BUILD}
 
-DEFINES -= GIT_VERSION=\"\\\"$$GIT_VERSION\\\"\"
-DEFINES += GIT_VERSION=\"\\\"$$CUSTOM_QGC_VERSION\\\"\"
+DEFINES -= APP_VERSION_STR=\"\\\"$$APP_VERSION_STR\\\"\"
+DEFINES += APP_VERSION_STR=\"\\\"$$CUSTOM_QGC_VERSION\\\"\"
```

Comment:
This does not work - build fails: main .pro/pri files still require `GIT_VERSION`. This works:
```suggestion
DEFINES -= GIT_VERSION=\"\\\"$$GIT_VERSION\\\"\"
DEFINES += GIT_VERSION=\"\\\"$$CUSTOM_QGC_VERSION\\\"\"
DEFINES -= APP_VERSION_STR=\"\\\"$$APP_VERSION_STR\\\"\"
DEFINES += APP_VERSION_STR=\"\\\"$$CUSTOM_QGC_VERSION\\\"\"
```